### ISSUE TITLE
[Bug] Gas-liquid Separator - Enthalpy for outlet streams equal zero

### ISSUE THREAD

**Issue by byryepez** [DanWBR/dwsim6#130]
IMPORTANT: This section is for code-related bugs. For usage issues, please go to the [Support Forums](https://sourceforge.net/p/dwsim/discussion/).

**Describe the bug**
During a calculation using the gas-liquid separator, the values for the enthalpies for the outlet streams (vapor and liquid) are equal to zero, while the same values for the feed line are not. The flash temperature was overridden to the inlet temperature, and outlet pressure is set to inlet minimum. Because the enthalpy on the vapor stream is zero, a valve cannot be calculated. Using a CAPE-OPEN package.

**Expected behavior**
Outlet streams should have the same value of enthalpy as the feed.

**Screenshots**
![image](https://user-images.githubusercontent.com/4457757/113764731-97464800-96d8-11eb-9674-4cd4a3b56a4e.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Version : 6.4.7

**Comment by DanWBR** [comment]
@byryepez Uncheck the "Skip equilibrium calculations..." option in the **Simulation Settings / Basis** panel:

<img width="891" alt="Captura de Tela 2021-04-07 às 17 09 00" src="https://user-images.githubusercontent.com/2669419/113934675-0b114f00-97c4-11eb-8172-0df9768aa281.png">

**Comment by byryepez** [comment]
Dear @DanWBR Daniel,

Thanks for checking this issue. I followed your suggestion and now the outlet streams show a value for the enthalpy. However, the outlet streams do not preserve the separation temperature (65°C), which affects the calculation of the outlet streams. 

![image](https://user-images.githubusercontent.com/4457757/113943351-c2f02d80-97bf-11eb-9a78-b03d54e1f492.png)

I use the same CAPE_OPEN package in COCO, and I obtained the following result:

![image](https://user-images.githubusercontent.com/4457757/113944521-e025fb80-97c1-11eb-8b5e-e2906c611642.png)

Is there any way to force the simulation to calculate the properties of the outlets based on T (or  P) and vapor fraction?

Thanks in advance.
